core/unit_test/CMakeLists.txt

```diff
@@ -539,7 +539,7 @@ KOKKOS_ADD_EXECUTABLE_AND_TEST(
 KOKKOS_ADD_EXECUTABLE_AND_TEST(
   UnitTest_CMakePassCmdLineArgs
   SOURCES UnitTest_CMakePassCmdLineArgs.cpp
-  ARGS one 2 THREE
+  ARGS "one 2 THREE"
```

Comment:
Why did you change this?

Reply:
@dalg24 this was done intentionally to match TriBITS behavior, here's link to the discussion comment
https://github.com/kokkos/kokkos/pull/3457#discussion_r501203687

Reply:
Yeah, this is a tribits issue. We have been following the rule that function `KOKKOS_X` has the exact same behavior as the `TRIBITS_X` function. I'd prefer not to break that rule.

Reply:
But according to the link to the documentation that was posted earlier https://tribits.org/doc/TribitsDevelopersGuide.html#tribits-add-test this was valid syntax

Reply:
Sorry, didn't explain well. You can pass multiple argument strings in, but that generates multiple tests. You create a test for entry in the list.